crates/ruff_linter/resources/test/fixtures/airflow/AIR001.py

```diff
@@ -1,16 +1,22 @@
 from airflow.operators import PythonOperator
+from airflow.providers.airbyte.operators.airbyte import AirbyteTriggerSyncOperator
+from airflow.providers.amazon.aws.operators.appflow import AppflowFlowRunOperator
 
 
 def my_callable():
     pass
 
 
 my_task = PythonOperator(task_id="my_task", callable=my_callable)
-my_task_2 = PythonOperator(callable=my_callable, task_id="my_task_2")
```

Comment:
cc @MichaReiser these two test cases are identical except that the arguments are reversed.